spec/pronto/github_spec.rb

```diff
@@ -124,5 +124,57 @@ module Pronto
         end
       end
     end
+
+    describe '#create_pull_request_review' do
+      subject { github.create_pull_request_review(comments) }
+
+      let(:octokit_client) { double(Octokit::Client) }
+
+      before do
+        github.instance_variable_set(:@client, octokit_client)
+      end
+
+      context 'with no comments' do
+        let(:comments) { [] }
+
+        specify do
+          octokit_client.should_not_receive(:create_pull_request_review)
+          subject
+        end
+      end
+
+      context 'with comments' do
+        before do
+          github.should_receive(:pull_id).once.and_return(pull_id)
```

Comment:
Is it OK to stub this method and not to deal with `ENV['PRONTO_PULL_REQUEST_ID']`?